doc/api/fs.md

```diff
@@ -2086,8 +2089,11 @@ an integer specifying the number of bytes to write.
 should be written. If `typeof position !== 'number'`, the data will be written
 at the current position. See pwrite(2).
 
-The callback will be given three arguments `(err, written, buffer)` where
-`written` specifies how many _bytes_ were written from `buffer`.
+The callback will be given three arguments `(err, bytesWritten, buffer)` where
+`bytesWritten` specifies how many _bytes_ were written from `buffer`.
+
+If this method is invoked as its [`util.promisify()`][]ed version, it returns
```

Comment:
Did https://github.com/addaleax/node/pull/5/commits/07acf080fc774cef2030b020e5f2a54d4016fb00 for timers, but not sure I can add an example here as there is no non-promise version here.